src/silo/query_engine/filter_expressions/has_mutation.cpp

```diff
@@ -27,7 +27,9 @@ std::unique_ptr<operators::Operator> HasMutation::compile(
    const silo::DatabasePartition& database_partition,
    AmbiguityMode mode
 ) const {
-   const char ref_symbol = database.reference_genome->genome_segments[0].at(position);
+   const char ref_symbol = database.reference_genomes.nucleotide_sequences
+                              .at(database.database_config.default_nucleotide_sequence)
+                              .at(position);
```

Comment:
This probably also needs to be generalized at some point?

Reply:
Same as below